src/Symfony/Component/CssSelector/Converter.php

```diff
@@ -19,11 +19,12 @@
 use Symfony\Component\CssSelector\XPath\Translator;
 
 /**
- * @author Christophe Coevoet <stof@notk.org>
+ * Converter is the main entry point of the component and can convert CSS
+ * selectors to XPath expressions.
  *
- * @api
+ * @author Christophe Coevoet <stof@notk.org>
  */
-class Converter implements ConverterInterface
+class Converter
```

Comment:
Maybe this should be called `Transformer`? As this transforms one convention to another ;)